config/initializers/statsd.rb

```diff
@@ -203,8 +203,17 @@
       params = Rack::Utils.parse_nested_query payload[:url].query
 
       if params['radius']
+        count = case payload.body
+                when Hash
+                  payload.dig(:body, :value) || 0
+                when Array
+                  payload.body&.count || 0
+                else
+                  0
+                end
```

Comment:
```suggestion
        count =
          case payload.body
          when Hash
            payload.body[:value]
          when Array
            payload.body.count
          end
```

Reply:
What do you think of moving the multiple `|| 0` down to where it's accessed?

Reply:
I'm worried about the & case